core/src/main/java/org/geysermc/geyser/impl/camera/GeyserCameraData.java

```diff
@@ -320,4 +321,17 @@ public void handleGameModeChange(boolean currentlySpectator, GameMode newGameMod
             }
         }
     }
+
+    /**
+     * Displays a notification toast for this connection.
+     *
+     * @param title the title of the toast.
+     * @param content the content of the toast.
+     */
+    public void sendToast(String title, String content) {
```

Comment:
the javadoc isn't necessary here tbh. the nonnull annotations can be here though.

also i would move this immediately below the hiddenElements() method